src/Nordea/Components/Modal.elm

```diff
@@ -58,6 +60,11 @@ newsModal msg =
     init msg NewsModal
 
 
+small : msg -> Modal msg
```

Comment:
Could we use withSmallSize so that it's more consistent with other components here?☺️

Reply:
Yes, I think that's a good idea. Will update